ci/setup_env.sh

```diff
@@ -4,12 +4,16 @@ set -eo pipefail
 IS_A_RELEASE="false"
 export PROD_RELEASE_BRANCH="${PROD_RELEASE_BRANCH:-master}"
 
+mix_version_tag="$(grep -oE 'version: "[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?(-RC[0-9]+)?"' mix.exs | grep -oE '[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?(-RC[0-9]+)?')"
```

Comment:
second grep may be grepping too much? It will grep all other versions
`$ grep -oE '[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?(-RC[0-9]+)?' mix.exs
5.2.1
0.3.0
0.1.0
1.5.3
1.7.0
0.30.1
1.0.3`

Reply:
at the moment the construct:
```
version: x.y.z
```
It is only in the version of the project, other dependencies don't contain the `version:` part.
So the first grep will pick:
```
version: x.y.z
```
and the second one will pick just the number from this subscript.
I tested it on the file and it works fine, but if you have any other suggestions on how to get that version number from the file I'll be happy to hear them 😄

Reply:
My bad. I have misread it as `||` vs `|`. I would use `cut` and `tr` command that is just smaller :)
```grep -oE 'version: "[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?(-RC[0-9]+)?"' mix.exs | cut -d ':' -f2 | tr -d '"'```

OR

```grep -oE 'version: "[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?(-RC[0-9]+)?"' mix.exs | cut -d ' ' -f2 | tr -d '"'```

But it is up to you.